src/battle_anim_new.c

```diff
@@ -9267,18 +9267,18 @@ void AnimTask_DynamaxGrowth(u8 taskId) // from CFRU
 
 void AnimTask_GetWeatherToSet(u8 taskId)
 {
-    switch (GetMoveMaxEffect(gCurrentMove))
+    switch (GetArgumentMoveEffect(gCurrentMove))
```

Comment:
This function will do nothing because the move effect was removed from `argMoveEffect`.